src/Process/3dTilesProcessing.js

```diff
@@ -144,8 +144,8 @@ export function process3dTilesNode(cullingTest, subdivisionTest) {
         if (node.parent.pendingSubdivision && !node.parent.additiveRefinement) {
             node.visible = false;
             // TODO: node.setDisplayed(false)?
-            if (node.material) {
-                node.material.visible = false;
+            if (node.content) {
```

Comment:
Maybe copy-paste the commit comment here. Or maybe add a `setTileVisible()` function, including this comment.
WDYT?